docs/source/cheat_sheet_py3.rst

```diff
@@ -109,8 +109,11 @@ Functions
        x.anything() + 1 + "string"  # no errors
 
    # This is how you annotate a callable (function) value
```

Comment:
Explanation on which values are the inputs of a function and which one is the return type of the function is a little ambiguous for newbies who are skimming through the cheatsheet.